zubhub_frontend/zubhub/public/index.html

```diff
@@ -47,6 +47,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Zubhub</title>
+    <script src="https://kit.fontawesome.com/58aa8d2b83.js" crossorigin="anonymous"></script>
```

Comment:
It would be nice if we can reuse some earlier and work and avoid adding new FA js

Reply:
Updated the PR